pkg/kube/util.go

```diff
@@ -155,6 +155,10 @@ func stringify(input interface{}) string {
 		{
 			return strconv.FormatBool(input.(bool))
 		}
+	case json.Number:
```

Comment:
Interesting, can you elaborate on how this is distinct from `Int`? I guess this code is written to assume data has been deserialized out of JSON into a struct so I don't understand what this is about

Reply:
This was necessary because of this https://github.com/IBM/argocd-vault-plugin/pull/90/files#diff-ef862b67a6c63e2e22fa26eb548c5c57f61388219beb2d7fee6ca1733af6542dR12.. i had a number in Vault and when it tried to do the replacement the type of the interface was `json.Number` instead of `Int` so i added this case to fix that error. The error was: 
```
panic: interface conversion: interface {} is json.Number, not string [recovered]
	panic: interface conversion: interface {} is json.Number, not string
```

Reply:
Hmmn OK